testing/test_HeMIS.py

```diff
@@ -1,89 +1,111 @@
 from ivadomed import losses
 import ivadomed.transforms as ivadomed_transforms
 
 cudnn.benchmark = True
 
 GPU_NUMBER = 0
-BATCH_SIZE = 8
+BATCH_SIZE = 4
 DROPOUT = 0.4
 BN = 0.1
 N_EPOCHS = 10
 INIT_LR = 0.01
-PATH_BIDS = 'testing_data/'
+PATH_BIDS = 'testing_data'
+p = 0.0001
 
 
-def test_unet():
-    HeMIS = True
-    device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
-    cuda_available = torch.cuda.is_available()
-    if cuda_available:
-        torch.cuda.set_device(GPU_NUMBER)
-        print("Using GPU number {}".format(GPU_NUMBER))
-
+def test_HeMIS(p=0.0001):
+    print('[INFO]: Starting test ... \n')
     training_transform_list = [
         ivadomed_transforms.Resample(wspace=0.75, hspace=0.75),
-        ivadomed_transforms.ROICrop2D(size=[48, 48]),
+        mt_transforms.CenterCrop2D(size=[48, 48]),
         mt_transforms.ToTensor()
     ]
     train_transform = transforms.Compose(training_transform_list)
 
     train_lst = ['sub-test001']
-    contrasts = ['T2w', 'T2star']
-    ds_train = loader.BidsDataset(PATH_BIDS,
-                                  subject_lst=train_lst,
-                                  target_suffix=["_lesion-manual"],
-                                  roi_suffix="_seg-manual",
-                                  contrast_lst=contrasts,
-                                  metadata_choice="without",
-                                  contrast_balance={},
-                                  slice_axis=2,
-                                  transform=train_transform,
-                                  multichannel=contrasts,
-                                  slice_filter_fn=SliceFilter(filter_empty_input=True, filter_empty_mask=False))
-
-    ds_train = loader.filter_roi(ds_train, nb_nonzero_thr=10)
-
-    train_loader = DataLoader(ds_train, batch_size=BATCH_SIZE,
+    contrasts = ['T1w', 'T2w', 'T2star']
+
+    print('[INFO]: Creating dataset ...\n')
+    dataset = adaptative.HDF5Dataset(root_dir=PATH_BIDS,
+                                     subject_lst=train_lst,
+                                     hdf5_name='testing_data/mytestfile.hdf5',
+                                     csv_name='testing_data/hdf5.csv',
+                                     target_suffix=["_lesion-manual"],
+                                     contrast_lst=['T1w', 'T2w', 'T2star'],
+                                     ram=False,
+                                     contrast_balance={},
+                                     slice_axis=2,
+                                     transform=train_transform,
+                                     metadata_choice=False,
+                                     dim=2,
+                                     slice_filter_fn=utils.SliceFilter(filter_empty_input=True, filter_empty_mask=True),
+                                     canonical=True,
+                                     roi_suffix="_seg-manual",
+                                     target_lst=['T2w'],
```

Comment:
Why sometimes it is `T2w_seg-manual`, while sometimes it is `T2w`?

Reply:
Both work since it is `if "T2w" in "string"`, but it should be only the modality i'll standardize it :)